### PR TITLE
Support Django 1.8

### PR DESCRIPTION
**Pull request by mikek**
Support Django 1.8
- Fix and extend test suite (was not actually respecting exit status!)
- Add Django 1.8 support

Note: this addon is currently archived on aldryn (only existing projects will get updates). I've released this version (tagged as 0.1.4.2 in develop branch) in alpha channel. Not sure if we want to support and release it in stable though.

### CODE REVIEW

**Comment by mikek** [comment]
@mkoistinen @intellectronica Decision on a possible release is required.

File: test_requirements/django_16.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-r base.txt
+Django>=1.6.9,<1.7
+django-discover-runner
```
Comment (cyberstar) [inline test_requirements/django_16.txt]: don't think that you need discover runner,  should be already a part of Django itself.

File: test_requirements/django_17.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+-r base.txt
+Django>=1.7,<1.8
+django-discover-runner
```
Comment (cyberstar) [inline test_requirements/django_17.txt]: don't think that you need discover runner,  should be already a part of Django itself.

File: test_requirements/django_18.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+-r base.txt
+Django>=1.8,<1.9
+django-discover-runner
```
Comment (cyberstar) [inline test_requirements/django_18.txt]: same as above.

**Comment by cyberstar** [comment]
LGTM, but please check comments.

**Comment by mikek** [comment]
@cyberstar you're right. Copy-pasting is bad =(

**Pull request merged**